### DISCUSSION TITLE
Can we hear, voice call and write it as opus?

### DISCUSSION THREAD

**Discussion by New-dev0** [Rapptz/discord.py#7281] [Q&A]
• Is it possible to hear voice communication with discord py? 
• Or can we write it as on time opus/raw file? 

• Was searching on Google, and got one Discord.js lib, which allows that, and maybe , its using the same protocol to do that.

Thanks in Advance!

**Comment by imvaskel** [comment]
Currently, discord.py does not support receiving voice. There are a couple pr's open for this, but at this very moment, it is unsupported.

**Reply by New-dev0** [reply]
thanks, will wait for that :)
